packages/compass-e2e-tests/tests/auto-connect.test.ts

```diff
@@ -245,7 +245,7 @@ describe('Automatically connecting from the command line', function () {
       await browser.waitForConnectionResult('success');
       await browser.execute(() => {
         // eslint-disable-next-line @typescript-eslint/no-var-requires
-        require('electron').ipcRenderer.call('test:show-connect-window');
+        require('electron').ipcRenderer.invoke('test:show-connect-window');
```

Comment:
Similarly here, this is unrelated but was required for me to be able to run e2e tests at all